mak/copyimports.d

```diff
@@ -0,0 +1,30 @@
+import std.array, std.conv, std.file, std.getopt, std.path, std.stdio;
+import core.stdc.stdlib;
+
+void main(string[] args)
+{
+    // DigitalMars make passes long command line through this environment variable _CMDLINE
+    if (auto p = getenv("_CMDLINE"))
+        args = split(to!string(p));
+    else
+        args = args[1..$];
+
+    string importPath = absolutePath("import");
+    string srcPath = absolutePath("src");
```

Comment:
This assumes that the script is run from the root project, right ?

Wouldn't the following at the module level be more robust:
```D
immutable RootPath    = __FILE_FULL_PATH__.dirName();
immutable ImportPath   = RootPath.buildPath("import");
immutable SourceTarget = RootPath.buildPath("src");
```
?

Reply:
My initial version allowed the import and src path to be configured via getopt, allowing it to be used in similar situations The current version assumes a bit more about files being passed on the command line, while your proposed change fixes it to the current single use.

It seems a bit strange to pass relative paths on the command line, but then assume other src/import paths internally. I'm not too opinionated, though.